GitUI/UserControls/RevisionGrid/RevisionGridMenuCommands.cs

```diff
@@ -110,8 +110,8 @@ private IReadOnlyList<MenuCommand> CreateNavigateMenuCommands()
                 new MenuCommand
                 {
                     Name = "GotoMergeBaseCommit",
-                    Text = "Go to merge base commit",
-                    ToolTipText = "Go to the merge base commit (last common commit) between the currently checked out commit (HEAD) and the currently selected commit",
+                    Text = "Go to common ancestor (merge base)",
```

Comment:
The Git merge-base documentation names this "Best common ancestor".
`Text = "Go to common ancestor",
ToolTipText = "Go to the best common ancestor (merge base) commit (last common commit) between the currently checked out commit (HEAD) and the currently selected commit",`

Reply:
We have too many parenthesis in the tooltip, suggest rewording to:
```
"Go to the common ancestor (merge base) commit, which is a last common commit between the currently checked out commit (HEAD) and the currently selected commit"
```

Reply:
We have too many parenthesis in the tooltip, suggest rewording to:
```
"Go to the common ancestor (merge base) commit, which is a best common commit between the currently checked out commit (HEAD) and the currently selected commit"
```

Reply:
ok for me

Reply:
Agree that @RussKie's text is easier to read. Otherwise I think this PR is great 👍

Reply:
Changed the tooltip text to suggested version.

Reply:
I think I made a typo and it should be "._..which is **the** best common commit..._" 😕

Reply:
No problem, fixed.